### PR TITLE
Hanna Branch

### PR DESCRIPTION
**Pull request by Hgersten-hash**

### CODE REVIEW

File: app/templates/profile.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    <title>Document</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="/Users/hannahgersten/Library/Mobile Documents/com~apple~CloudDocs/MLHFellowship/Portfolio/project-team-pythonic/app/static/styles/profile.css"
```
Comment (jescalada) [inline app/templates/profile.html]: We should make the href relative to the project file structure to make sure it works anywhere.

**Review by jescalada — CHANGES_REQUESTED** [review]

File: app/templates/profile.html (hunk 1 above)
Comment (jescalada) [reply app/templates/profile.html]: Made a change to the href, feel free to merge!

**Review by jescalada — APPROVED** [review]

**Comment by EmilieYZhang** [comment]
Great work! There was just a javascript link issue in profile.html, Juan will be fixing up. Everything else looks good.

**Comment by jescalada** [comment]
I fixed the javascript filepath issue and also restructured the scripts folder to be inside the static folder

**Review by EmilieYZhang — APPROVED** [review]
Everything works now! Great job, looks amazing.

**Pull request merged**